modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the type of storage provided by your selected storage class. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.
+
+For example:
+
+* When using Ceph RBD, your access mode is automatically set to `ReadWriteMany`, which improves efficiency during Live Migration, among other performance benefits.
+* File system overhead is automatically calculated for you. You will receive at least two gigabytes of usable space to contain the virtual machine disk.
```

Comment:
fs overhead is chosen automatically (if required)

Reply:
Think I covered that in the revision above.